www/_template/guides.md

```diff
@@ -101,16 +33,16 @@ background: linear-gradient(30deg, #A7BFE8, #6190E8);
   Popular Integration Guides
 </h3>
 
-<div class="news-items">
+<div class="card-grid card-grid-4">
 {% for post in collections.communityGuide %}
 
-<article class="news-item">
+<article class="card">
   <a href="{{ post.url }}" style="text-decoration: none; color: initial; flex-grow: 1;">
```

Comment:
The `<a>` tag here may need a class, because in some cases it doesn’t take up `width: 100%`:

<img width="733" alt="Screen Shot 2021-01-25 at 15 07 43" src="https://user-images.githubusercontent.com/1369770/105772229-1839d400-5f1f-11eb-8572-dac315f7a4ad.png">

This causes the thumbnail to have some empty space on the right. But just adding `width: 100%` to this tag should fix it

Reply:
Also, nesting `<div>`s and `<p>`s and `<h3>`s inside a parent anchor tag can cause issues, both from a CSS standpoint (at a certain point, the browser won’t render things properly), as well as from an a11y standpoint. This CSS Tricks article sums it up: [Block Links Are a Pain (and Maybe Just a Bad Idea)](https://css-tricks.com/block-links-are-a-pain-and-maybe-just-a-bad-idea/)